app/src/main/java/ru/chaichuk/hwapp/api_v3/MovieDbApi.kt

```diff
@@ -0,0 +1,66 @@
+package ru.chaichuk.hwapp.api_v3
+
+import com.jakewharton.retrofit2.converter.kotlinx.serialization.asConverterFactory
+import kotlinx.serialization.json.Json
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.http.GET
+import retrofit2.http.Path
+import retrofit2.http.Query
+import ru.chaichuk.hwapp.HWApp
+import ru.chaichuk.hwapp.api_v3.dto.CreditsDTO
+import ru.chaichuk.hwapp.api_v3.dto.MovieDTO
+import ru.chaichuk.hwapp.api_v3.dto.MovieDetailsDTO
+import ru.chaichuk.hwapp.api_v3.dto.MoviesListPageDTO
+
+class MovieDbApi {
```

Comment:
why you need this class which doing nothing

Reply:
I think, I used it to hide all retrofit staff inside that class

Reply:
okay, but it looks like just proxy which doing nothing